libogc/timesupp.c

```diff
@@ -261,11 +261,13 @@ static u32 __getRTC(u32 *gctime)
 time_t time(time_t *timer)
 {
 	time_t gctime = 0;
+	u32 rtctime;
 #if defined(HW_RVL)
 	u32 wii_bias = 0;
 #endif
 
-	if(__getRTC((u32*)&gctime)==0) return (time_t)0;
+	if(__getRTC(&rtctime)==0) return (time_t)0;
+	gctime += rtctime;
```

Comment:
Why `+=` instead of just `=`?

Reply:
Brainfart, mostly. I was changing it from a version where `gctime` might have been non-zero and didn't realize that the `+` wasn't necessary anymore when I committed.